sink/src/it/java/com/datastax/oss/kafka/sink/simulacron/SimpleEndToEndSimulacronIT.java

```diff
@@ -430,12 +434,12 @@ void failure_offsets() {
         .contains("Connector has no configuration for record topic 'wrongtopic'")
         .contains("Could not parse 'bad key'")
         .contains(
-            "statement: INSERT INTO ks1.table1(a,b) VALUES (:a,:b) USING TIMESTAMP :kafka_internal_timestamp");
+            "statement: INSERT INTO ks1.table1(a,b) VALUES (:a,:b) USING TIMESTAMP :"
+                + SinkUtil.TIMESTAMP_VARNAME);
     InstanceState instanceState = task.getInstanceState();
-    assertThat(instanceState.getFailedRecordCounter("mytopic", "ks1.table1").getCount())
-        .isEqualTo(3);
-    assertThat(instanceState.getRecordCounter("mytopic", "ks1.table1").getCount()).isEqualTo(4);
-    assertThat(instanceState.getFailedWithUnknownTopicCounter().getCount()).isEqualTo(1);
+    assertThat(instanceState.getFailedRecordCounter("mytopic", "ks1.table1")).isEqualTo(3);
```

Comment:
I changed this method in "commons" to return a "long" because in PulsarSink we cannot use Metrics dependency in ITs due to classpath clash.